runtime/src/main/java/com/flipkart/flux/deploymentunit/DeploymentUnit.java

```diff
@@ -112,4 +112,14 @@ public Object getObjectMapperInstance() {
     public Object getInjectorClassInstance() {
         return injectorClassInstance;
     }
+
+    public Map<String, Object> getTaskConfiguration(String taskPrefix) {
+        Iterator<String> allKeys = configuration.getKeys("taskConfig." + taskPrefix);
```

Comment:
instead of getting keys and removing "taskConfig.taskPrefix", you can use configuration.subset("taskConfig.taskPrefix") which returns Configuration object with keys not having the prefix.